components/Nav/nav.js

```diff
@@ -73,6 +86,26 @@ export default function NavBar() {
                             </a>
                         </Link>
                     ))}
+                    <ul
+                        className="w-11/12 sm:w-1/4 flex list-none mt-3 mb-4 flex-row min-w-fit rounded-full box-border border-2 border-white bg-gray-200"
```

Comment:
Does anyone have a suggestion on how to update the classes so it aligns properly?